docs/reference/msbuild-targets.md

```diff
@@ -464,6 +465,41 @@ msbuild -t:restore -p:RestorePackagesConfig=true
 > [!NOTE]
 > `packages.config` restore is only available with `MSBuild 16.5+`, and not with `dotnet.exe`
 
+### Restoring with MSBuild static graph evaluation
+
+> [!NOTE]
+> With MSBuild 16.6+, NuGet has added an experimental feature to use static graph evaluation from the commandline that significantly decreases the restore time for large repositories.
```

Comment:
```suggestion
> With MSBuild 16.6+, NuGet has added an experimental feature to use static graph evaluation from the command line that significantly improves the restore time for large repositories.
```

Reply:
Good catch. "improvement" tends to be used more for perf related work, so let's use that.